api/token.go

```diff
@@ -200,7 +200,7 @@ func (a *API) ResourceOwnerPasswordGrant(ctx context.Context, w http.ResponseWri
 		}
 
 		if cookie != "" && config.Cookie.Duration > 0 {
```

Comment:
Based on the conversation last night with @thorwebdev, we always want to set the cookie and Thor suggested removing this check. Is this check needed?

Reply:
Sounds like from @thorwebdev in the above comment, the duration and cookie check can be removed here, too.

Reply:
I only removed it from the /verify endpoint because we can't pre-emptively set the header for GET requests. The /token endpoint is a POST request which means we can set the x-use-cookie header always.

Reply:
Yep, I think that's fine. I do think there isn't any harm in always setting the cookie across the board though if we wanted to align it across routes.